Procfile

```diff
@@ -1 +1 @@
-web: gunicorn -b 0.0.0.0:$PORT -w $GUNICORN_WORKER_COUNT "vulnerable_people_form:create_app(None)"
+web: gunicorn -b 0.0.0.0:$PORT -w $GUNICORN_WORKER_COUNT "vulnerable_people_form:create_app(None)" -c gunicorn_config.py
```

Comment:
is this gunicorn_config.py still relevant ? As the file has a single import just referencing the child_exit function

Reply:
I think so. That is part of the setup for the metrics library. 

https://github.com/alphagov/gds_metrics_python